myuw/templates/handlebars/card/registration/reg_resources.html

```diff
@@ -82,15 +82,15 @@ <h4 class="sr-only">Registration resources</h4>
             </li>
         {{ /if }}
 
-	{{ #if is_pce }}
+	{{ #if is_c2 }}
```

Comment:
PCE Time schedule should be displayed for C2 student. Place the PCE Time schedule at top of the resources link.